db/seeds.rb

```diff
@@ -8,3 +8,4 @@
 require File.dirname(__FILE__) + '/seed_helper.rb'
 
 create_roles_and_permissions
+populate_languages
```

Comment:
I'd take a different approach:
- leave the seed task as it is (perhaps include the task to fill the languages)
- include this rake task in a cap task that executes each time a deploy happens
then all bases are covered.